
<launch>

  <arg name="model" default="$(find vss_simulation)/urdf/vss_robot.xacro" />

  <!-- Spawn robot #0 -->
  <group ns="robot_0">
    <param name="tf_prefix" value="robot0_tf" />
    <include file="$(find vss_simulation)/launch/spawn_robot.launch" >
      <arg name="robot_number"  value="0" />
      <arg name="pose" value="-x 1 -y 1 -z 0" />
      <arg name="model" value="$(arg model)" />
    </include>
  </group>

  <!-- Spawn robot #1 -->
  <group ns="robot_1">
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find vss_simulation)/launch/spawn_robot.launch" >
      <arg name="robot_number"  value="1" />
      <arg name="pose" value="-x 0 -y 1 -z 0" />
      <arg name="model" value="$(arg model)" />
    </include>
  </group>

  <!-- Spawn robot #2 -->
  <group ns="robot_2">
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find vss_simulation)/launch/spawn_robot.launch" >
      <arg name="robot_number"  value="2" />
      <arg name="pose" value="-x 0 -y 0 -z 0" />
      <arg name="model" value="$(arg model)" />
    </include>
  </group>

</launch>